### PR TITLE
Issue 20892 update docker clean

### LINKED ISSUE #20892: Updates to gzip compression settings in server.xml Docker template
1) In the [Docker server.xml template](https://github.com/dotCMS/core/blob/master/docker/dotcms/ROOT/srv/templates/tomcat/OVERRIDE/conf/server-9.0.41.xml), gzip compression is enabled on ports 8080 and 8443 Connectors only. 
It should be enabled on all ports (cloud uses 8082 extensively).

```
compression="on"
compressionMinSize="128"             
compressableMimeType="text/html,..."
useSendfile="false"
```

2)  `compressableMimeType` should include fonts and svg
```
font/eot
font/otf
font/ttf
image/svg+xml
```

3) Port 8081 Connector can probably be removed. 
Cloud does not use or need 8081.

### LINKED ISSUE #21454: Clean up docker image
We need to clean up how our docker image runs.  This includes

- [x] removing `dockerize` and the dockerized templates 
- [x] removing the init/waiting scripts for db and elasticsearch
- [x] using only dotCMS environmental variables to configure docker.  This means that for the most part, configuring the DB, ES, and dotCMS options should be the same whether you are running locally or in docker.
- [x] Removing hazelcast customizations - if hazelcast is to be used, admins should mount a custom hazelcast.xml file
- [x] Centralizing all config variables into a `config-defaults.sh`
- [x] Allow for APR to init and run
- [x] Incorporate the changes from the other docker issues, e.g.: 
     - #21440
     - #21063
     - #20892

### PR DESCRIPTION
**Pull request by wezell**
This updates our docker image to use standard environmental variables for configuration.

### CODE REVIEW

**Review by nollymar — APPROVED** [review]

File: docker/dotcms/ROOT/srv/40-custom-starter-zip.sh [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
 set -e
 
+
+
 CUSTOM_STARTER=custom_starter.zip
 
 ## if we have a custom starter
 if [ -z ${CUSTOM_STARTER_URL} ]; then
-	echo "Using default starter"; 
+	echo "";
```
Comment (victoralfaro-dotcms) [inline docker/dotcms/ROOT/srv/40-custom-starter-zip.sh:11]: Is something need to be echoed here?

File: docker/dotcms/ROOT/srv/00-config-defaults.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+export DOT_ES_AUTH_JWT_TOKEN=${DOT_ES_AUTH_JWT_TOKEN:-""}
+export DOT_ES_ENDPOINTS=${DOT_ES_ENDPOINTS:-"https://es.dotcms.site:9200"}
+
+## if you want to provide a custom starter for the initial data load, specify
+# a url here and dotCMS will download it before starting up
+export CUSTOM_STARTER_URL=${CUSTOM_STARTER_URL:-""}
+
+## This needs to be set in order for catalina to read environmental properties
+export CATALINA_OPTS="$CATALINA_OPTS -Dorg.apache.tomcat.util.digester.PROPERTY_SOURCE=org.apache.tomcat.util.digester.EnvironmentPropertySource"
+
```
Comment (victoralfaro-dotcms) [inline docker/dotcms/ROOT/srv/00-config-defaults.sh:83]: Liking this very much

File: docker/dotcms/ROOT/srv/40-custom-starter-zip.sh (hunk 1 above)
Comment (nollymar) [reply docker/dotcms/ROOT/srv/40-custom-starter-zip.sh]: Thanks! I've rolled back the message

**Review by victoralfaro-dotcms — APPROVED** [review]
LGTM

**Pull request merged**